### DISCUSSION TITLE
Use an existing SQLITE database?

### DISCUSSION THREAD

**Discussion by mihkuno** [pocketbase/pocketbase#3811] [Q&A] [answered]
Its my first time using sqlite realizing they're stored as a database file 

I experimented by making my own `data.db` with records in it using the `DB Browser for SQLITE` and replaced `data.db` 

Also disabled automigration. But it didn't seem to work, is there an option to make it work or is this question weird?

**Comment by ganigeorgiev** [comment, marked as answer]
This is a valid question but at the moment it is not supported.

You can open and explore the db file just fine, but when editing records by hand they need to satisfy the PocketBase fields structure and the records table must have a `_collections` entry reflecting the fields schema.

A more "native" SQLite development experience is planned before final v1 (including tables auto discovery and allowing loading custom SQLite amalgamation) but other tasks are with higher priority at the moment.

**Reply by dschissler** [reply]
I just want to say that Pocketbase is almost unusable because it doesn't have a complete [bubbletea](https://github.com/charmbracelet/bubbletea) front end for server debugging.
